setup.py

```diff
@@ -69,25 +69,23 @@
     ],
     extras_require = {
         'docs':  [
-            # XXX: Intersphinx URLs are broken when using non-default inventory
-            # (like in Django's case) with Sphinx 1.4.3+
-            # Bug report: https://github.com/sphinx-doc/sphinx/issues/2789
-            'Sphinx==1.4.2',
+            'Sphinx>=1.4.6',
             'sphinx_rtd_theme',
         ],
         'package': [
             'twine',
             'wheel',
         ],
         'test': [
-            'coverage>=3.7.1',
-            'pep8>=1.6.2',
-            'pylint>=1.4.3',
-            'testfixtures>=4.10.0',
             'check-manifest',
+            'coverage>=4.2',
+            'pycodestyle>=2.0.0',
+            'pydocstyle>=1.0.0',
+            'pylint>=1.6.4',
             'readme_renderer',
             'resolwe-runtime-utils>=1.1.0',
-            'Sphinx==1.4.2',
+            'Sphinx>=1.4.6',
```

Comment:
@mstajdohar, thanks for the explanation, I forgot about the autoprocess Sphinx extension.

I agree with your last comment, we should move it to `install_requires` since we explicitly import Sphinx from [`resolwe/flow/utils/docs/autoprocess.py`](https://github.com/genialis/resolwe/blob/master/resolwe/flow/utils/docs/autoprocess.py#L39).